backend/triday_utils.py

```diff
@@ -0,0 +1,10 @@
+import math
+
+async def get_triday(day):
```

Comment:
Don't need async here or on the next function as it serves no purpose and causes issues in the other file (I made the same mistake before)